core/assets/sounds/cues/marchingOrdersEn/data.json

```diff
@@ -0,0 +1,212 @@
+{
+  "gameID": "marchingOrdersEng",
```

Comment:
Please refer to the [language guidelines](https://github.com/chrislo27/RhythmHeavenRemixEditor2/wiki/Guidelines-and-Syntactical-Sugar#language-guidelines). Use `marchingOrdersEn` (you did this in data.json which is right but not in here for some reason).